pages/common/bw.md

```diff
@@ -0,0 +1,23 @@
+# bw
+
+> A CLI to access and manage a Bitwarden vault.
+
+- To login to a Bitwarden user account:
+
+`bw login`
+
+- To logout of a Bitwarden user account:
+
+`bw logout`
+
+- To search and display items from Bitwarden vault:
+
+`bw list items --search {{github}}`
+
+- To display a particular item from Bitwarden vault:
+
+`bw get item {{github}}`
+
+- To create a folder in Bitwarden vault:
```

Comment:
```suggestion
- Create a folder in Bitwarden vault:
```

Also, just out of interest, does it always require a base64-encoded JSON string as input? 🙂

Reply:
Also, is it necessary for the text to be in json ?

Reply:
Hmm, looks like it does. :thinking_face:

https://help.bitwarden.com/article/cli/#create